src/components/AssetsTable/index.jsx

```diff
@@ -368,41 +537,87 @@ export class AssetsTable extends React.Component {
 
   renderModal() {
     return (
-      <Modal
-        open={this.state.modalOpen}
-        title={`Delete ${this.state.assetToDelete.display_name}`}
-        body={this.renderModalBody()}
-        closeText="Cancel"
-        onClose={this.closeModal}
-        buttons={[
-          <Button
-            label="Permanently delete"
-            buttonType="primary"
-            onClick={this.deleteAsset}
-          />,
-        ]}
-        variant={{ status: Variant.status.WARNING }}
-      />
+      <WrappedMessage message={messages.assetsTableCancel} >
+        {
+          displayText => (
+            <Modal
+              open={this.state.modalOpen}
+              title={(<WrappedMessage
+                message={messages.assetsTableDeleteObject}
+                values={{
+                  displayName: this.state.assetToDelete.display_name,
+                }}
+              />)}
+              body={this.renderModalBody()}
+              closeText={displayText}
```

Comment:
This is where I landed on the `closeText` issue. The entire modal is enclosed in a containing `<span lang='en'>`, whose language code will match the rendered language of the "Cancel" string. There are other messages included in the Modal (which could potentially be in a different language); they'll have their own containing spans. The cancel button body and `×` aria-label are the only text that relay on the parent Modal's span:

![screen shot 2018-02-01 at 10 27 36 am](https://user-images.githubusercontent.com/7373924/35686987-9454023e-073b-11e8-907b-c32f1803f64b.png)
<img width="1003" alt="screen shot 2018-02-01 at 10 34 49 am" src="https://user-images.githubusercontent.com/7373924/35686988-94665434-073b-11e8-984c-a43c470498b0.png">